src/main/java/org/jenkinsci/plugins/inodesnodemonitor/InodesMonitor.java

```diff
@@ -73,7 +70,7 @@ public Object data(Computer computer) {
 			String computerName = computer.getName();
 			// master has no nodeName
 			if ("".equals(computer.getName())) {
-				computerName = "master";
+				computerName = "built-in";
```

Comment:
I recommend the `getSelfLabel()` method instead of using a hard coded string that differs depending on the Jenkins version.
```suggestion
				computerName = computer.getNode().getSelfLabel();
```